Steps4Impact.xcodeproj/project.pbxproj

```diff
@@ -1479,7 +1479,7 @@
 				CODE_SIGN_ENTITLEMENTS = "Steps4Impact/Supporting Files/Steps4Impact.entitlements";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 272;
+				CURRENT_PROJECT_VERSION = 273;
 				DEVELOPMENT_TEAM = 6SQ8F32998;
```

Comment:
Note this build number is of the last release. Wasn't committed to master so added it.